### ISSUE TITLE
[Manage tokens - QA]  UI allows changing the vote when the voting has finished

### ISSUE THREAD

**Issue by olgapod** [JoinColony/colonyCDapp#3603]
## Description: 
The issue is that you shouldn't be allowed to click on "Change vote", as your first vote actually finished the voting phase. So the UI should not show you the option to Change vote anymore

## Steps to reproduce

1. Install the reputational Weight extension.
2. Open  manage tokens Actions
3. Add one token, fill in all fields and submit the form.
4. Go to the vote step and Support vote.
5. Change vote and oppose the vote.

https://www.figma.com/design/5V8pr7iMwXsT9L3VAZsmUt/Actions-%26-Motions?node-id=3615-356361&t=RydvhxIe7fdiiH1T-0

## Expected behaviour

* UI should not show you the option to Change your vote.

## Actual behaviour

* UI show the option to Change vote on the Vote step of manage tokens action.

https://github.com/user-attachments/assets/da262495-abda-4550-921d-ce902378121d

Uploading 51.mp4…

**Comment by jakubcolony** [comment]
Right, now that I can see the recording, I understand what's going on better.

After you voted, the vote threshold was met, and the motion immediately advanced into the reveal phase. The UI wrongly gave you the option to change your vote, which you managed to trigger just before it updated - but this was never going to work on the contracts.

So the actual issue here is that the UI allows changing the vote when the voting has finished, which happens when the vote takes the motion past the threshold.

**Comment by rdig** [comment]
@olgapod please update the issue with the new specifications as described by Jakub above

**Comment by olgapod** [comment]
Thanks for comments, I changed the issue
